packages/grid/veilid/veilid.dockerfile

```diff
@@ -11,14 +16,17 @@ RUN cd veilid-server && cargo build --release -p veilid-server
 # ========== [Stage 2] Dependency Install ========== #
 
 FROM python:3.12-bookworm
```

Comment:
I initially planned to use wolfi, since, we would be shifting to using veilid pre-built binaries instead of building from source (We build from source currently, as there is a small bug in the official pre-built binaries)
example commit: [f820db1](https://github.com/OpenMined/PySyft/pull/8526/commits/f820db1de23d8de4ffdbd3cf8e44ddd58f52c027)

I guess veilid would not be available in wolfi apk packages, and we would have to add it to wolfi by their request template due to which, we cannot use veilid in wolfi at the moment
![Screenshot 2024-03-12 at 2 52 44 PM](https://github.com/OpenMined/PySyft/assets/43314053/b0c19278-ad22-4a14-82bf-ae01776d5b38)

Not sure If I am fully correct here.

Reply:
+1, also can we pass python 3.12 as ARG instead